comms.py

```diff
@@ -0,0 +1,62 @@
+from mcu import *
+from controls import *
```

Comment:
This is a cyclic import. Since controls lies above comms on the hierarchy, don't import it again; instead, rely on methods to communicate between classes.